.github/workflows/nightly-fuzz-test.yml

```diff
@@ -38,8 +38,13 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           RUST_MIN_STACK: 8388608
 
+      - name: Get current date
+        id: date
+        run: echo "DAY=date::$(date +'%u')" >> $GITHUB_OUTPUT
+
       - name: Send an alert in slack
-        if: ${{ failure() }}
+        # Send notifications on weekdays if there's a failure.
+        if: ${{ failure() && steps.date.outputs.DAY <= 5 }}
```

Comment:
Would be nice to still get an alert for failures that happened over the weekend on Monday.

Reply:
Chances are that they'll fail again on monday tbh. I checked on slack and there doesn't seem to be a way to delay these messages until a particular time.